tests/unit/test_rapu.py

```diff
@@ -154,3 +157,21 @@ def test_content_type_re():
         "serialization_format": "json",
         "general_format": None,
     }
+
+
+async def test_raise_connection_error_handling() -> None:
+    request_mock = Mock()
```

Comment:
**[non-blocking]**
Specifying `spec` could aid understanding of what is mocked directly in the test.
I see there's a `rapu.HTTPRequest`, but that doesn't have a `.read()` method.

Reply:
Done.